### DISCUSSION TITLE
LearnXinYminutes

### DISCUSSION THREAD

**Discussion by NetLancer** [vlang/v#11813] [Ideas]
Hi guys! Just wanted to say i'd love to see v-lang documentation appear in learnxinyminutes.com, cos i used to find some new and modern polished languages there myself. And probably it would benefit its popularity over time a bit. 
I like v-lang and want to see it growing and maturing for production cases. 
Thanx !:))

**Comment by JalonSolov** [comment]
Perhaps after the 1.0 release.  Current is 0.2.2, and the language is still changing... a document such as you mention would be changing too much to be useful unless someone follows the language closely, until it stabilizes.

**Comment by litenite42** [comment]
We actually have code setup here [V Community](https://github.com/v-community/learn_v_in_y_minutes). I'm just not familiar w/ Ruby to get the Site running to make sure it matches.

**Comment by litenite42** [comment]
It needs some touch up to talk on sum types, interfaces, and a few other topics. Otherwise, it's probably mostly good. I do want to rewrite some of the examples so they don't vary from addresses to vehicles to fruits as you go through, but only so much time lol
